### DISCUSSION TITLE
Getting a lot of gif.part## and gif's that don't work.

### DISCUSSION THREAD

**Discussion by cscott0108** [MalloyDelacroix/DownloaderForReddit#361] [Q&A]
Getting an issue when downloading where I see a bunch of gif.part## files and whole .gif files that don't work... Not sure if this is an issue with the downloader but I've seen the parts go up to 40 so it says file_name.gif.part40... is there a way to collapse these parts into 1 workable file or do I need to delete them. Also, I have a few .gif files that do not load in dedicated photo viewers like photoshop or ACDSee Photo Studio as it is being unrecognized. Any way to fix these files?

**Comment by MalloyDelacroix** [comment]
I believe the parts you are seeing are due to the multi-part download function downloading the gifs in parts, but failing to combine the gifs at the end.  To resolve this, open the main settings menu, and uncheck "Multi-Part Download".
